content/applications/general/users/access_rights.rst

```diff
@@ -52,6 +52,48 @@ The :guilabel:`Administration` field in the :guilabel:`Access Rights` tab has th
 .. image:: access_rights/user-permissions-dropdown-menu.png
    :alt: The Sales apps drop-down menu to set the user's level of permissions.
 
+Manage specific permissions
+---------------------------
+
+While access rights are typically managed in a bundle, they can also be set for explicit
+permissions. For example, giving a user the :guilabel:`Administrator` permission for **Timesheets**
+gives them full access to that app. That user, however, can be made to have full access to the app,
+but their ability to manage their own timesheets restricted (e.g., if the payroll administrator is
+salaried and does not need to manage their own timesheets).
+
+Manage specific permissions requires :ref:`developer mode <developer-mode>`. After that, navigate to
+:menuselection:`Settings app`, and then click :guilabel:`Manage users`, select a user, and go to the
+:guilabel:`Technical Access Rights` tab. From here, groups can be edited (which is covered in the
+next section) and specific access rights can be managed.
+
+In the :guilabel:`Groups` section, these specific access rights are organized by groups and are
+split into sections. If no changes are made to these groups, then their permissions will mirror the
+selections made in the :guilabel:`Access Rights` tab.
+
+- :guilabel:`Selected groups`: access rights set by choices made in the :guilabel:`Access Rights`
+  tab
+- :guilabel:`Groups added automatically`: access rights given as part of the access rights provided
+  by the :guilabel:`Selected groups`
+
+To add a permission to this user profile, click :guilabel:`Add a line` in the :guilabel:`Selected
+groups` table, and then add permissions to this user profile. To remove a permission, click the
+:icon:`fa-times` :guilabel:`(cancel)` at the end of that permission's row.
```

Comment:
Would mention here that _removing_ permissions may have further permission removal consequences in the **Groups added automatically** table, since one informs the other. Maybe a warning block would be a useful format to emphasize this?